base/db/R/rename_jags_columns.R

```diff
@@ -18,18 +18,21 @@
 rename_jags_columns <- function(data) {
   
   # Change variable names and calculate obs.prec within data frame
+  # Swap column names
+  colnames(data)[colnames(data) %in% c("greenhouse", "ghs")] <- c("ghs", "greenhouse")
+  colnames(data)[colnames(data) %in% c("site_id", "site")] <- c("site", "site_id")
```

Comment:
Can you explain a bit why this is needed? On a quick read it looks as if these columns get swapped upstream in `jagify` and then swapped back here, but I'm not sure what happens to them in between.

Reply:
Apologies for the confusion. Basically, I am trying to retain both the original name of the identifying column and the new name (the id given by BETYdb and the index assigned in the meta-analysis). They get swapped back because of the column names required downstream by pecan.ma().

Reply:
That makes sense -- I didn't think to check downstream. 

It might be useful to add a comment here to that effect for future maintainers, but I'll approve this now so that my change request isn't blocking.